modules/navier_stokes/doc/content/source/fvkernels/NSFVFrictionFlowDiode.md

```diff
@@ -0,0 +1,13 @@
+# NSFVFrictionFlowDiode
```

Comment:
This naming brings up the issue of how we name our classes. On our table [here](https://mooseframework.inl.gov/modules/navier_stokes/index.html) we don't have any explanation for `NSFV`. This currently inherits from `INSFVElementalKernel`, but perhaps you're hesitant to call this `INSFVFrictionFlowDiode` because you want it be seen as usable for `INSFV`, `PINSFV`, `WCNSFV`, and `PWCNSFV`? Which makes total sense. However, with the transition to the action, my thought is that our naming convention for classes can be seen more as reflections of implementation, instead of input file use ... in which case I think `INSFV` is the most appropriate since it's meant only to be applied for pressure-based implementations, there is no requirement for a fluid properties object such as for a weakly compressible treatment (though our implementation for *that* is apparently controversial 😄 ), and there is no requirement for a porosity functor. What do you think?

Reply:
INSFV it is. I think that s what we have been doing so far